### PR TITLE
gaia-flex: Add big-dipper-setting.json

### PR DESCRIPTION
**Pull request by orkunkl**
Adds required settings for big-dipper configuration.
Related to https://github.com/CosmWasm/testnets/issues/57

### CODE REVIEW

File: gaia-flex/big-dipper-settings.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                "fraction": 1000000
+            }, 
+            {
+                "denom": "umuon",
+                "displayName": "MUON",
+                "displayNamePlural": "MUONS",
+                "fraction": 1000000
+            } 
+        ],
+        "gasPrice": 0.025,
```
Comment (orkunkl) [inline gaia-flex/big-dipper-settings.json:25]: is this value fit for node's gas price as well?

Comment (ethanfrey) [reply gaia-flex/big-dipper-settings.json:25]: yup

**Review by ethanfrey — APPROVED** [review]
Looks good

File: gaia-flex/big-dipper-settings.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "slashingWindow": 10000,
+        "uptimeWindow": 250,
+        "initialPageSize": 30,
+        "bech32PrefixAccAddr": "cosmos",
+        "bech32PrefixAccPub": "cosmospub",
+        "bech32PrefixValAddr": "cosmosvaloper",
+        "bech32PrefixValPub": "cosmosvaloperpub",
+        "bech32PrefixConsAddr": "cosmosvalcons",
+        "bech32PrefixConsPub": "cosmosvalconspub",
+        "bondDenom": "uatom",
```
Comment (ethanfrey) [inline gaia-flex/big-dipper-settings.json]: Wait... this is `umuon`

File: gaia-flex/big-dipper-settings.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "powerReduction": 1000000,
+        "coins": [
+            {
```
Comment (ethanfrey) [inline gaia-flex/big-dipper-settings.json:18]: No atoms here... remove this section please

**Review by ethanfrey — COMMENTED** [review]
Okay, too fast the first time. Minor adjustments to token types.

File: gaia-flex/big-dipper-settings.json (hunk 2 above)
Comment (orkunkl) [reply gaia-flex/big-dipper-settings.json]: I got these values from [Default big-dipper](https://github.com/forbole/big-dipper/blob/master/default_settings.json) settings assuming these are the values that are used in cosmos mainnet. Are you sure? maybe these coin representations used internally by big-dipper? @ethanfrey

**Pull request merged**